### PR TITLE
chore(readme): update readme with preferred install procedure

### PR DESCRIPTION
**Pull request by lofgrenfredrik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,13 @@
 
 ## Install
 
+This config and it's dependencies are included in We aheads tooling packages and don't need to be installed in new We ahead projects. Instead use one of the tooling packages that have all configs and dependencies needed for it's use.
```
Comment (michaellopez) [inline README.md]: @lofgrenfredrik `needed for it's use` => `needed for its use`

Comment (michaellopez) [reply README.md]: @lofgrenfredrik `and it's dependencies` => `and its dependencies`

Comment (michaellopez) [reply README.md]: @lofgrenfredrik `We aheads` => ``We ahead`s``

File: README.md [hunk 2]
```diff
@@ -4,6 +4,13 @@
...
+
+- For **non-React** projects use [@weahead/tooling](https://github.com/weahead/tooling)
+- For **React** projects use [@weahead/tooling-react](https://github.com/weahead/tooling-react)
+
+### Install the config by it's own
```
Comment (michaellopez) [inline README.md]: @lofgrenfredrik `it's` => `its`

**Review by michaellopez — CHANGES_REQUESTED** [review]

**Comment by lofgrenfredrik** [comment]
@michaellopez
fixed copy. 👍

**Review by michaellopez — APPROVED** [review]

**Pull request merged**